CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [7.1.0]
+### Added
+- Add CAIP-2 and CAIP-10 Types ([#116](https://github.com/MetaMask/utils/pull/116))
```

Comment:
```suggestion
- CAIP-2 and CAIP-10 Types ([#116](https://github.com/MetaMask/utils/pull/116))
```

Reply:
All the other Added sections start their list with `Add` 🥲